projects/plugins/crm/admin/email/main.page.php

```diff
@@ -159,7 +159,7 @@ function jpcrm_render_emailbox() {
 				// echo '<img class="ui avatar image" src="/images/avatar/small/helen.jpg">';
 					echo '<div class="content">';
 					echo '<div class="header">' . esc_html( $email->zbsmail_subject ) . '</div>';
-						echo '<div class="the_content">' . esc_html( zeroBSCRM_io_WPEditor_DBToHTMLExcerpt( $email->zbsmail_content, 200 ) ) . '</div>';
+						echo '<div class="the_content">' . esc_html( wp_html_excerpt( $email->zbsmail_content, 200 ) ) . '</div>';
```

Comment:
Didn't know about `wp_html_excerpt()`! Do we still need `esc_html()` in this context?

Reply:
Shouldn't hurt either way. Looks like WP core has one instance where they use it:

https://github.com/WordPress/WordPress/blob/1d561e1fa6b8f965e039dd18f7029d41296af240/wp-admin/async-upload.php#L66

And several instances where they don't.

Reply:
Ah this is partly to appease PHPCS, which suggests 'All output should be run through an escaping function ' for this line. And yes, like you spotted it also appears to be in use in WordPress core 😄  The main difference appears to be that `esc_html` also converts some additional special characters into their HTML entities, particularly quotes.
Could add an ignore for PHPCS but also like you say it doesn't hurt in this case with the extra escape function.